test/config/arrow-function-parameters/main.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @minItems 1
+ */
+export type MyNonEmptyArray<T> = T[];
+/**
```

Comment:
Let's not have too many comments here as it makes the test more complicated than necessary.

Reply:
Simplified in 82e27e2